src/img2catalog/xnat_parser.py

```diff
@@ -35,7 +37,7 @@ def __init__(self, message: str, error_list: List[str] = None):
         self.error_list = error_list
 
 
-def xnat_to_DCATDataset(project: XNATBaseObject, config: Dict) -> DCATDataSet:
+def xnat_to_DCATDataset(project: XNATBaseObject, config: Dict) -> Tuple[DCATDataset, URIRef]:
```

Comment:
would it be possible to apply the same approach as for the catalog?

Reply:
I'm not sure if that approach is desirable, as then I'd have to maintain two separate lists and xnat_list_datasets would still have to return two separate lists, one for the DCATDataset object and one for the URIs.